doc/faq.rst

```diff
@@ -1,6 +1,29 @@
 Frequently Asked Questions about Autocrypt
 ==========================================
 
+Why is there no verification mechanism?
```

Comment:
better "key verification" let's not be too implicit ... Saying "key verification" already hints at a conflict with Autocrypt's stated design philosophy, btw: "Don't talk to users about keys, ever". Just saying :)

Reply:
is this actually stated somewhere as a design philosophy? we can get around a lot of "key" language, but I don't really think the term itself is the core of the problem - explaining asymmetric crypto is.

anyways, agreed about the not too implicit :+1: